test/qunit/model/FHIRModel.unit.js

```diff
@@ -1143,4 +1144,27 @@ sap.ui.define([
 		}.bind(this));
 		assert.ok(bMatch, "The filter matched one of the given values for dummy property");
 	});
+
+	QUnit.test("Get Updated Resource from response when location has / or not shouldnot fail ", function(assert) {
+		var sResourceId = this.oFhirModel1.create("Patient", {
+			name: [
+				{
+					family: "james",
+					given: ["parker"]
+				}
+			]
+		});
+		var sPatientPath = "/Patient/" + sResourceId;
+		this.oFhirModel1.bindContext(sPatientPath);
```

Comment:
Why we need the call of bindContext here? In general, testing private methods its not intended. 
The test scenario 1 could look like following:
1. Create Model with dummy data
2. Simulate GET request and return location header without slash
3. Test if resource is stored correctly

The test scenario 2 could look like following:
1. Create Model with dummy data
2. Simulate GET request and return location header with slash
3. Test if resource is stored correctly

Reply:
@flovogt : i have tried approach as suggested. let me know if its fine..

Reply:
@flovogt In one of the response entry I have added / in location and tested without the fix and the test cases are failing